projects/uswds-components/src/lib/textarea/textarea.component.html

```diff
@@ -0,0 +1,7 @@
+<textarea [ngClass]="{'usa-input--error': showError, 'usa-input--success': !showError && control && control.valid}"
+  [(ngModel)]="model" type="textarea" class="usa-textarea" [attr.id]="name" [attr.placeholder]="placeholder" #textarea
```

Comment:
We should use the id property from the ts file for `[attr.id]` attribute, and set `name` to the `name` attribute of this text area?

Reply:
Updated